src/components/Leaderboard/Leaderboard.tsx

```diff
@@ -160,7 +160,10 @@ function PaginatedItems() {
                               className={styles.pic}
                               src={getAvatarByID(row.user.avatarId).url}
                             ></img>
-                            {' ' + row.user.username}
+                            {' ' +
+                              (row.user.username.length > 100
+                                ? row.user.username.substring(0, 10) + '...'
+                                : row.user.username)}
```

Comment:
- Always have first 10 char 
```suggestion
                                row.user.username.substring(0, 10)
```
- Do the similar update in `DC-LeaderBoard` and `BattleTv`